Python/cim-convert-tool/configSshCim17.py

```diff
@@ -0,0 +1,191 @@
+        },
+        'cim:EquivalentInjection': {
+            'mainAttributes': {
+                'rdf:about': {'type': 'string', 'list': False} # @id
+            },
+            'tags': {
+                'cim:Equipment.inService': {'type': 'bool', 'list': False},
+                'cim:EquivalentInjection.p': {'type': 'float', 'list': False, 'CIMDatatype': 'cim:ActivePower.value'},
+                'cim:EquivalentInjection.q': {'type': 'float', 'list': False, 'CIMDatatype': 'cim:ReactivePower.value'}
+            },
+            'attributes': {
+
+            }
+        },
+        'cim:Fuse': {
+            'mainAttributes': {
+                'rdf:about': {'type': 'string', 'list': False} # @id
+            },
+            'tags': {
+                'cim:Equipment.inService': {'type': 'bool', 'list': False},
+                'cim:Switch.locked': {'type': 'bool', 'list': False},
+                'cim:Switch.open': {'type': 'bool', 'list': False}
+            },
+            'attributes': {
+
+            }
+        },
+        'cim:GeneratingUnit': {
+            'mainAttributes': {
+                'rdf:about': {'type': 'string', 'list': False} # @id
+            },
+            'tags': {
+                'cim:Equipment.inService': {'type': 'bool', 'list': False}
+            },
+            'attributes': {
+
+            }
+        },
+        'cim:LinearShuntCompensator': {
+            'mainAttributes': {
+                'rdf:about': {'type': 'string', 'list': False} # @id
+            },
+            'tags': {
+                'cim:Equipment.inService': {'type': 'bool', 'list': False}
+            },
+            'attributes': {
+
+            }
+        },
+        'cim:LoadBreakSwitch': {
+            'mainAttributes': {
+                'rdf:about': {'type': 'string', 'list': False} # @id
+            },
+            'tags': {
+                'cim:Equipment.inService': {'type': 'bool', 'list': False},
+                'cim:Switch.locked': {'type': 'bool', 'list': False},
+                'cim:Switch.open': {'type': 'bool', 'list': False}
+            },
+            'attributes': {
+
+            }
+        },
+        'cim:PetersenCoil': {
+            'mainAttributes': {
+                'rdf:about': {'type': 'string', 'list': False} # @id
+            },
+            'tags': {
+                'cim:Equipment.inService': {'type': 'bool', 'list': False}
+            },
+            'attributes': {
+
+            }
+        },
+        'cim:PowerTransformer': {
+            'mainAttributes': {
+                'rdf:about': {'type': 'string', 'list': False} # @id
+            },
+            'tags': {
+                'cim:Equipment.inService': {'type': 'bool', 'list': False}
+            },
+            'attributes': {
+
+            }
+        },
+        'cim:SynchronousMachine': {
+            'mainAttributes': {
+                'rdf:about': {'type': 'string', 'list': False} # @id
+            },
+            'tags': {
```

Comment:
RotatingMachine.p, RotatingMachine.q, RegulatingCondEq.controlEnabled CIM attributes and SynchronousMachine.operatingMode reference are missing (used in MV file)